### ISSUE TITLE
Damp cloth smothering

### ISSUE THREAD

**Issue by Lumipharon** [tgstation/-tg-station#6077]
So damp rags can hold 5u of a reagent in them, and if you use the damp rag on a person, you get both a smothering message, and a wiping down message.

I assume, that the idea of this is to transfer the reagent to the person, but instead it applies it to them.
This has the hilarious effect of you being able to set smokers on fire with a alcohol soaked rag, but means you can't smother someone with smash/chloral to ghetto stun them. I assume that it also means you could probably dip it in acid and wreck people, with acid in it's current state, but hey.

TL:DR, damp rag is supposed to transfer reagents to a person (smothering them) as opposed to slathering them up with the reagent.
